docs/source/docker/getting_started/first_steps.rst

```diff
@@ -640,11 +640,10 @@ The output directory is structured in the following way:
 * data:
    The data directory contains everything to do with data. 
    
-    * If `enable_corruptness_check=True`, it will contain a "clean" version of the dataset. 
-    * If `remove_exact_duplicates=True`, it will contain a copy of the `embeddings.csv` 
-      where all duplicates are removed. Otherwise, it will simply store the 
-      embeddings computed by the model.
-    * If `selected_sequence_length > 1`, it will contain a `sequence_information.json`
+    * `embeddings.csv` contains the computed embeddings for all new input samples used in selection.
+    * `selected_embeddings_including_datapool.csv` contains the embeddings of all selected samples.
```

Comment:
> of all selected samples

including the ones that were preselected because they are in the datapool.

Reply:
done